examples/210-embedding_models.ipynb

```diff
@@ -24,7 +24,8 @@
     "    * [Baseline](#section_2_1)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    ts.df_exog = None
</pre><p>Looks strange... Can we avoid doing this? </p><p>dE.g. we can recreate tsdataset using to_pandas. I just think it isn't good to show users they can change df_exog this way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>